fades/file_options.py

```diff
@@ -0,0 +1,94 @@
+# Copyright 2014-2016 Facundo Batista, Nicolás Demarchi
+#
+# This program is free software: you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 3, as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranties of
+# MERCHANTABILITY, SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR
+# PURPOSE.  See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# For further info, check  https://github.com/PyAr/fades
+
+"""Parse fades options from confir files."""
+
+import logging
+from configparser import ConfigParser
+
+logger = logging.getLogger(__name__)
+
+CONFIG_FILES = ("/etc/fades.ini", "~/.fades.ini", ".fades")
```

Comment:
I don't know what to do with the /etc files...   I will find if I find a solution